packages/suite-desktop/src-electron/electron.ts

```diff
@@ -346,13 +346,23 @@ app.on('browser-window-focus', (_event, win) => {
     }
 });
 
+let bridgeStartThrottle = null;
 ipcMain.on('bridge/start', async (_event, devMode?: boolean) => {
+    if (bridgeStartThrottle) {
+        return;
+    }
+
     try {
         if (devMode) {
             await bridge.startDev();
         } else {
             await bridge.start();
         }
+
+        // Throttle bridge start
```

Comment:
Is throttling bridge-specific or should we abstract this into BaseProcess?

Reply:
The front-end is calling `bridge/start` multiple times until the bridge starts, so I guess we can say it's bridge specific for now. I'd like to fix that in the front-end (as well/instead) but as I was already working on the back-end part, I fixed it there :smile:

Reply:
Ah, okay. In that case, let's leave it here :)